actions/apt_action.go

```diff
@@ -72,9 +80,15 @@ func (apt *AptAction) Run(context *debos.DebosContext) error {
 	if err != nil {
 		return err
 	}
-	err = c.Run("apt", "apt-get", "clean")
-	if err != nil {
-		return err
+	/* 
+	 * Keep packages for further processing if they weren't
+	 * installed immediately
+	 */
+	if !apt.Downloadonly {
+		err = c.Run("apt", "apt-get", "clean")
```

Comment:
what if another apt action is run after with downloadonly set to false ? It will run apt-get clean ?

Reply:
Indeed, the way I imagined things:
- run apt with `downloadonly`
- immediately copy packages elsewhere
- you're free to do whatever you want afterwards

This will probably cause multiple issues along the way and is bound to confuse users, so I'll rework this so the "copy packages elsewhere" bit is also handled in the same action.

Any suggestion about the options in this case? Would a single `downloadonly: <path>` option work, or would you prefer both `downloadonly: true` and `downloadto: <path>`?

Reply:
`downloadonly: <path>` seems more straightforward